manifests/ros-image-demo/deployment.yaml

```diff
@@ -1,22 +1,20 @@
 name: ros-image-demo
-toolchainRegion: us-west-2
+toolchainRegion: eu-west-1
 groups:
   - name: optionals
     path: manifests/ros-image-demo/optional-modules.yaml
   - name: core
     path: manifests/ros-image-demo/core-modules.yaml
   - name: docker-images
     path: manifests/ros-image-demo/docker-image-modules.yaml
-  - name: analysis
-    path: manifests/ros-image-demo/aws-analysis-modules.yaml
-  - name: ford-analysis
-    path: manifests/ros-image-demo/ford-analysis-modules.yaml
+  - name: dags
+    path: manifests/ros-image-demo/image-pipeline-modules.yaml
 targetAccountMappings:
   - alias: primary
     accountId: 1234567890
     default: true
     parametersGlobal:
       dockerCredentialsSecret: aws-addf-docker-credentials
     regionMappings:
-      - region: us-west-2
```

Comment:
what will be the impact of CI by this change (if any)?

Reply:
reverted